packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -929,7 +926,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
          * of this client's connection - https://github.com/microsoft/FluidFramework/issues/8375.
          */
         const getCurrentTimestamp = () => {
-            return this.deltaManager.lastMessage?.timestamp ?? performance.now();
+            return this.deltaManager.lastMessage?.timestamp ?? Date.now();
```

Comment:
I believe we prefer to use performance.now() where performance comes from common-utils

Reply:
peformance.now() returns a time relative to the document load. So, it will not be comparable to the timestamp in ops which is the number of milliseconds since the unix epoch.
I could do something like `performance.timing.navigationStart + performance.now()` to get a comparable timestamp but performance.timing is deprecated.